trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
+  end
+end
```

Comment:
- [ ] Great job working on the decorator class but here in the trimmer class you have something to add a condition and to make sure  that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word. as mentioned in the requirements 🔥🤩💯🚀 

- So you should check first it is longer than 10 then trim the word ✅
![image](https://github.com/tajemouti/school-library/assets/58553711/e1a7dfc3-911c-4b93-a2e4-7e11f276ac8f)

Reply:
Hi @Reem,
Thank you for your review and feedback. I appreciate your attention to detail. I understand your point, and I've carefully reviewed the code.
The code indeed fulfills the requirement of truncating the name to a maximum of 10 characters when it exceeds that length. It does so by using the slice method.
However, I acknowledge your feedback, and I'm willing to make the suggested change, even though the code is already functioning correctly in this regard. Your feedback is valuable, and I want to ensure that the code aligns with the intent of the requirements.
I will update the code as per your suggestion to make it even more explicit. Thanks again for the review, and I appreciate your help in improving the code.
Cheers